# HG changeset patch
# User jwe
# Date 744858224 0
#      Mon Aug 09 01:03:44 1993 +0000
# Node ID a2db6687354d3d77c105d91d36891ed7b6dfe7bd
# Parent  70f622a3d197e1e44de2905d3dbd5a3872a600a4
[project @ 1993-08-09 01:03:44 by jwe]
Include defaults.h.

diff --git a/src/utils.cc b/src/utils.cc
--- a/src/utils.cc
+++ b/src/utils.cc
@@ -81,16 +81,17 @@ extern "C"
 #include "procstream.h"
 #include "user-prefs.h"
 #include "variables.h"
 #include "error.h"
 #include "utils.h"
 #include "octave.h"
 #include "mappers.h"
 #include "version.h"
+#include "defaults.h"
 #include "tree-const.h"
 #include "unwind-prot.h"
 #include "octave-hist.h"
 
 // Top level context (?)
 extern jmp_buf toplevel;
 
 // Pipe to gnuplot.
